data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -535,11 +544,20 @@ else
     fi
 fi
 
-# Check if the API and API_INT Server URLs can be reached.
-echo "Check if API and API-Int URLs are reachable during bootstrap"
+if [ ! -f api-dns-check.done ]; then
+  echo "Check if API URL is reachable during bootstrap"
 
-check_url "API_URL" "${API_SERVER_URL}"
-check_url "API_INT_URL" "${API_INT_SERVER_URL}"
+  if check_url "API_URL" "${API_SERVER_URL}"; then
+    touch api-dns-check.done
+  fi
+fi
+
+if [ ! -f api-int-dns-check.done ]; then
+  echo "Check if API-Int URL is reachable during bootstrap"
+  if check_url "API_INT_URL" "${API_INT_SERVER_URL}"; then
+    touch api-int-dns-check.done
```

Comment:
Nit: this file could be called <url type>-reachable.done.

Will not hold PR for this nit.